integration/hurl/tests_ok/netrc_option.hurl

```diff
@@ -0,0 +1,6 @@
+GET http://localhost:8000/netrc
+
```

Comment:
Can we remove this blank line (just a personal `flavor` for the moment, we'll try to enforec the format of Hurl files later)